lucid/misc/io/reading.py

```diff
@@ -93,6 +95,12 @@ def read_handle(url, cache=None, mode="rb"):
     """
     url = scope_url(url)
 
+    if isazure(url):
+        handle = blobfile.BlobFile(url, mode)
+        yield handle
+        handle.close()
+        return
```

Comment:
Nice!

Do we want these to be cached, and if so, would it make sense to migrate this down to the `if cache:/else:` block below? If we don't want caching, then it makes sense here.

Reply:
good catch. forgot about caching!